cli/client.go

```diff
@@ -380,6 +387,83 @@ func RobotsPartShellAction(c *cli.Context) error {
 	)
 }
 
+// checkUpdateResponse holds the values used to hold release information.
+type checkUpdateResponse struct {
+	Name       string `json:"name"`
+	TagName    string `json:"tag_name"`
+	TarballURL string `json:"tarball_url"`
+}
+
+// CheckUpdateAction is the corresponding Action for 'check-update'.
+func CheckUpdateAction(c *cli.Context) error {
+	if c.Bool(quietFlag) {
+		return nil
+	}
+
+	dateCompiledRaw := rconfig.DateCompiled
+	dateCompiled, err := time.Parse("2006-01-02", dateCompiledRaw)
+	if err != nil {
+		warningf(c.App.ErrWriter, "CLI Update Check: failed to parse compiled date: %w", err)
+		return nil
+	}
+
+	// compare to current version
+	appVersion := rconfig.Version
+
+	// skip check if stable & install is older than two week
+	if appVersion != "" && time.Since(dateCompiled) < time.Hour*24*7*2 {
+		return nil
+	}
```

Comment:
comment doesn't match

```suggestion
	// skip check if stable and install is less than 2 weeks old
	if appVersion != "" && time.Since(dateCompiled) < time.Hour*24*7*2 {
		return nil
	}
```